File: app/src/main/java/com/jobinlawrance/pics/MainActivity.java
package com.jobinlawrance.pics;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.jobinlawrance.pics.home.HomeFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .add(R.id.fragment_container, new HomeFragment())
                    .commit();
        }
    }
}

File: app/src/main/java/com/jobinlawrance/pics/MainActivity.kt
package com.jobinlawrance.pics

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.jobinlawrance.pics.home.HomeFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .add(R.id.fragment_container, HomeFragment())
                    .commit()
        }


    }
}
